src/components/tables/EntityTable.tsx

```diff
@@ -108,10 +130,11 @@ function EntityTable({
     useEffect(() => {
         if (selectData && selectData.length > 0) {
             setTableState({ status: TableStatuses.DATA_FETCHED });
+            enableSelection ? setRows(selectData) : null;
```

Comment:
Hiding some calls behind a flag for allowing selection. These could happen just fine since there are stores always wrapping but eventually we want this stuff to not always run.

Reply:
This is an instance where an if statement would be fine, as opposed to a ternary. Not a _necessary_ adjustment.